src/api/service.ts

```diff
@@ -0,0 +1,19 @@
+import { refreshAccessToken } from "api/auth"
+import { logOut, setAccessToken } from "utils"
+
+export async function refreshAccessTokenAction() {
```

Comment:
No need to create an Action in a separate service file for api functions.
You can use it directly in the interceptor logic.